include/fmt/std.h

```diff
@@ -13,6 +13,7 @@
 #include <cstdlib>
 #include <exception>
 #include <memory>
+#include <source_location>
```

Comment:
Extra (unconditional) include, please remove.

Reply:
Sorry about that, forgot to remove it when I added the conditional include.